src/Microsoft.PowerShell.Commands.Utility/commands/utility/UtilityCommon.cs

```diff
@@ -238,16 +238,19 @@ public override string ToString()
                     // Display each byte, in 2-digit hexadecimal, and add that to the left-hand side.
                     nextLine.AppendFormat("{0:X2} ", currentByte);
 
-                    // If the character is printable, add its ascii representation to
-                    // the right-hand side.  Otherwise, add a dot to the right hand side.
-                    if ((currentByte >= 0x20) && (currentByte <= 0xFE))
-                    {
-                        asciiEnd.Append((char)currentByte);
-                    }
-                    else
-                    {
-                        asciiEnd.Append('.');
+                    // Add its ascii representation to the right-hand side.
+                    // If the byte represents a control character, replace with a space for byte value 0   
+                    // and /uFFFD for all others.
+                    char charToAppend = (char)currentByte;
+                    if (char.IsControl(charToAppend)) {
+                        // 
+                        if (currentByte == 0) {
+                            charToAppend = ' ';
+                        } else {
+                            charToAppend = '\uFFFD';
```

Comment:
Can you comment on what character this is?

Reply:
@TravisEz13 Additional comments added as requested.